experimental/dds/tree2/src/feature-libraries/optional-field/optionalField.ts

```diff
@@ -34,18 +31,16 @@ import {
 	RemovedTreesFromChild,
 } from "../modular-schema";
 import { nodeIdFromChangeAtom } from "../deltaUtils";
-import { OptionalChangeset, OptionalFieldChange } from "./optionalFieldChangeTypes";
+import { RegisterId, OptionalChangeset } from "./optionalFieldChangeTypes";
 import { makeOptionalFieldCodecFamily } from "./optionalFieldCodecs";
 
-type ChangeId = ChangeAtomId | "self";
-
 interface IChildChangeMap<T> {
```

Comment:
Consider renaming to `IRegisterMap` or `IRegistry` since it is now used for more than just child changes.
Same for `ChildChangeMap`